CMakeLists.txt

```diff
@@ -93,6 +101,7 @@ else() # Windows MinGW!
 	add_compile_options(
     -Wno-uninitialized
     -w
+    $<$<COMPILE_LANGUAGE:Fortran>:-cpp>
     $<$<COMPILE_LANGUAGE:Fortran>:-std=legacy>
     $<$<COMPILE_LANGUAGE:Fortran>:-fno-second-underscore>
 	  $<$<COMPILE_LANGUAGE:Fortran>:-fallow-argument-mismatch>
```

Comment:
Probably good to fix the indentation here, this looks like a merge accident or something.

Reply:
There was a tab, which was not seen in VS code. Fixed